.github/workflows/ci_env.yml

```diff
@@ -0,0 +1,46 @@
+name: Lisflood OS Unit Tests
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - name: Checkout github repo
+      uses: actions/checkout@v3
+    - uses: conda-incubator/setup-miniconda@v2
```

Comment:
I would recommend using micro mamba instead, it's much much faster!
I used it there for instance:
https://github.com/ecmwf/hat/blob/develop/.github/workflows/ci.yml
Also, why don't you use the requirements file instead of installing everything one by one? This would be a way to test the requirements file as well

Reply:
I temporarily removed the action, it will be added on next release.